src/mds/Server.cc

```diff
@@ -2892,15 +2892,13 @@ void Server::handle_client_open(MDRequestRef& mdr)
 
   int flags = req->head.args.open.flags;
   int cmode = ceph_flags_to_mode(flags);
-
-  bool need_auth = !file_mode_is_readonly(cmode) || (flags & O_TRUNC);
-
-  dout(7) << "open on " << req->get_filepath() << dendl;
```

Comment:
We want this output line prior to quitting so that it shows up in debug logs. Please don't move it!